src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,91 @@
-export const Pagination = () => {};
+import { getNumbers } from "../../utils";
+import cn from "classnames";
+
+type Props = {
+  currentPage: number;
+  items: string[];
+  perPage: number;
+  onPageChange: (page: number) => void;
+};
+
+export const Pagination: React.FC<Props> = ({
+  currentPage,
+  items,
+  perPage,
+  onPageChange,
+}) => {
+  const handlePrevPage = () => {
+    if (currentPage > 1) {
+      onPageChange(currentPage - 1);
+    }
+  };
+
+  const handleNextPage = () => {
+    if (currentPage < Math.ceil(items.length / perPage)) {
+      onPageChange(currentPage + 1);
+    }
+  };
+
+  return (
+    <>
+      <ul className="pagination">
+        <li
+          className={cn('page-item', {
+          disabled: currentPage === 1,
+          })}
+        >
+          <a
+            data-cy="prevLink"
+            className="page-link"
+            href="#prev"
+            onClick={handlePrevPage}
+            aria-disabled={currentPage === 1 ? 'true' : 'false'}
+          >
+            «
+          </a>
+        </li>
+        {getNumbers(1, Math.ceil(items.length / perPage)).map(num => (
+          <li
+            key={num}
+            className={cn('page-item', {
+              active: currentPage === num,
+            })}
+            onClick={() => onPageChange(num)}
+          >
+            <a data-cy="pageLink" className="page-link" href={`#${num}`}>
```

Comment:
The `href` attribute for pagination links could lead to unwanted URL changes. Consider using a button or preventing the default event action if these are meant to be SPA (Single Page Application) links.